### PR TITLE
Quickbooks Integration V4

### PR DESCRIPTION
**Pull request by Shwetabhk**

### CODE REVIEW

**Comment by Shwetabhk** [comment]
Screenshot 2020-08-28 at 4 38 29 PM
Screenshot 2020-08-28 at 4 38 47 PM
Screenshot 2020-08-28 at 4 38 56 PM
Screenshot 2020-08-28 at 4 39 10 PM
Screenshot 2020-08-28 at 4 39 25 PM
Screenshot 2020-08-28 at 4 39 40 PM

File: src/app/qbo/expense-groups/expense-groups.component.html [hunk 1]
```diff
@@ -33,12 +33,12 @@
 <div *ngIf="count" [hidden]="isLoading">
   <table mat-table [dataSource]="expenseGroups" class="page-table" matSort matSortActive="created_at"
     matSortDisableClear matSortDirection="desc">
-    <ng-container matColumnDef="description">
+    <!-- <ng-container matColumnDef="description">
```
Comment (ashwin1111) [inline src/app/qbo/expense-groups/expense-groups.component.html:36]: why why

File: src/app/qbo/expense-groups/expense-groups.component.html [hunk 2]
```diff
@@ -33,12 +33,12 @@
...
       <th mat-header-cell *matHeaderCellDef> Description </th>
       <td mat-cell *matCellDef="let element">
         Grouped by {{element.fyle_group_id}}
       </td>
-    </ng-container>
+    </ng-container> -->
```
Comment (ashwin1111) [inline src/app/qbo/expense-groups/expense-groups.component.html:41]: why why

File: src/app/qbo/qbo.component.html [hunk 3]
```diff
@@ -25,7 +25,7 @@
       <a class="mat-h3 qbo-sidenav--nav-item" mat-list-item>
         Settings
       </a>
-      <a class="mat-h3 qbo-sidenav--nav-sub-item" mat-list-item routerLink="{{workspace.id}}/settings/configurations/"
+      <a class="mat-h3 qbo-sidenav--nav-sub-item" mat-list-item routerLink="{{workspace.id}}/settings/configurations/general"
```
Comment (ashwin1111) [inline src/app/qbo/qbo.component.html:28]: general/

**Review by ashwin1111 — APPROVED** [review]

File: src/app/qbo/settings/configuration/general-configuration/general-configuration.component.ts [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+        },
+        {
+          label: 'Check',
+          value: 'CHECK'
+        }
+        ];
+      }
+
+      that.generalSettingsForm.controls.employees.disable();
+      that.generalSettingsForm.controls.reimburExpense.disable();
```
Comment (Sravanksk) [inline src/app/qbo/settings/configuration/general-configuration/general-configuration.component.ts:95]: `reimburExpense.disable();` --> `reimbursableExpense.disable();`

File: src/app/qbo/settings/configuration/general-configuration/general-configuration.component.html [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+          </mat-option>
+        </mat-select>
+      </mat-form-field>
+
+      <div class="configurations--label">
+        Map reimbursable Expenses from Fyle to
+      </div>
+      <mat-form-field floatLabel="always" appearance="outline" required>
+        <mat-select placeholder="Choose Quickbooks equivalent of Reimbursable Expenses"
+          formControlName="reimburExpense">
```
Comment (Sravanksk) [inline src/app/qbo/settings/configuration/general-configuration/general-configuration.component.html:37]: `reimburExpense.disable();` --> `reimbursableExpense.disable();`

**Review by Sravanksk — APPROVED** [review]

**Pull request merged**